ArchiSteamFarm/www/js/app.js

```diff
@@ -122,20 +122,22 @@ function logCommand(state, cmd) {
     var currentdate = new Date();
     var datetime = currentdate.getDate() + "."
```

Comment:
Tip: If you want to show the current time in Console, you don't have to use this. Just enable an option called `Show timestamps` in the Developer Tools' preferences:

![image](https://user-images.githubusercontent.com/35700659/35520190-7a227ebc-04fd-11e8-9903-d8085f01709b.png)


To modify your preferences, open the Developer Tools and hit `F1` (works on Chrome at least).